### DISCUSSION TITLE
Win10 RTL-SDR - no support. GNU Radio?

### DISCUSSION THREAD

**Discussion by bogus105** [jopohl/urh#873] [Q&A] [answered]
I was trying to find issue #139 as metioned here in comments:
https://www.rtl-sdr.com/reverse-engineering-signals-universal-radio-hacker-software/comment-page-1/#comment-216211

speciffically:

> February 22, 2017 - 8:50 am
Jerome
The author tolds on the issues board of the project, that RTL-SDR is not natively supported. One must use a GnuRadio backend (see bug #139). Any volounteers is needed to add native rlt-sdr support !

Please direct me how to make URH work on my Win10 64-bit.
Thank you and thank you for the great piece of software!

**Comment by jopohl** [comment]
This comment is very old, RTL-SDR works of the box on Windows natively. Just use the exe installer on the releases page: https://github.com/jopohl/urh/releases/tag/v2.9.2

**Comment by bogus105** [comment]
Thank you for reply. On mine it's ont working out of the box. On my machine RTL-SDR option is grayed out. It is not enabled. Kindly see attached.
![urh](https://user-images.githubusercontent.com/28808389/118375440-f5145c80-b5c1-11eb-867f-f3b0ff8e4064.png)

**Comment by jopohl** [comment]
On the screenshot you posted RTL-SDR is enabled through native backend...

**Comment by bogus105** [comment]
If so i don't understand why it's not working when i want to record any signal? Why is shows "not available"?

**Comment by jopohl** [comment]
Please make a screenshot.

**Comment by bogus105** [comment]
I'm getting eeror code -12:

![urh1](https://user-images.githubusercontent.com/28808389/118393304-bffa1f80-b63e-11eb-9ce5-98ad1d2ad336.png)

**Comment by jopohl** [comment, marked as answer]
So the native support **is** working. This error code comes from the RTL-SDR driver. Did you google it? Some things that come to my mind:
1. Try run URH as administrator
2. Try a different USB port
3. Not sure if this is needed for RTL-SDR but for HackRF you need to install driver with Zadig. Might be worth a shot.

**Reply by bogus105** [reply]
Bull's eye! Point number 1. Run as administrator! Now it's working. Thank you.
